config/crd/bases/operator.open-cluster-management.io_multiclusterhubs.yaml

```diff
@@ -195,16 +195,16 @@ spec:
                 - failedProvisionConfig
                 type: object
               hubSize:
-                description: The resource allocation bucket for this hub to use.
-                  [S (Small), M (Medium), L (Large), XL (Extra Large)]. Defaults to
-                  (M)edium if not specified.
-                type: string
+                default: Medium
+                description: The resource allocation bucket for this hub to use. [S
+                  (Small), M (Medium), L (Large), XL (Extra Large)]. Defaults to (M)edium
+                  if not specified.
                 enum:
-                  - S
-                  - M
-                  - L
-                  - XL
-                default: M
+                - Small
+                - Medium
+                - Large
+                - ExtraLarge
+                type: integer
```

Comment:
I'm not sure if we want to have this as an integer. If the default is `Medium`, I don't think this would be valid in the CRD. If we need to, we can remove the `enum` from the CRD and just use `default: Medium` as a baseline.

Reply:
no, this passes the tests. The type only defines how big it is, but enums are typically represented as integers in most software languages